docs/content/error/$controller/ctrlreg.ngdoc

```diff
@@ -0,0 +1,24 @@
+@ngdoc error
+@name $controller:ctrlreg
+@fullName A controller with this name is not registered.
+@description
+
+This error occurs when the {@link ng.$controller `$controller()`} service is called
+with a string that does not match any of the registered controllers. The controller service may have
+been invoked directly, or indirectly through the {@link ng.ngController `ngController`} directive,
```

Comment:
I would add "for example through...", because the list is non-exhaustive.
(E.g. a route definitions for `ngRoute` or `ui.router` and possibly `ngMaterial` instantiate controllers as well under the hood.)